src/KafkaFlow.Retry.UnitTests/KafkaFlow.Retry/Durable/Polling/QueueTrackerTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿namespace KafkaFlow.Retry.UnitTests.KafkaFlow.Retry.Durable.Polling
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using global::KafkaFlow.Retry.Durable.Definitions;
+    using global::KafkaFlow.Retry.Durable.Polling;
+    using Moq;
+    using Quartz;
+    using Xunit;
+
+    public class MockIJob : IJob
+    {
+        public Task Execute(IJobExecutionContext context)
+        {
+            var jobExecutionContexts = context.JobDetail.JobDataMap["JobExecution"] as List<IJobExecutionContext>;
+            jobExecutionContexts.Add(context);
+
+            return Task.CompletedTask;
+        }
+    }
+
+    public class QueueTrackerTests
+    {
+        [Fact]
+        public async Task QueueTracker_Create_Success()
```

Comment:
I think we can explain better what we're trying to achieve with this test by setting the name as:
`QueueTracker_ScheduleAndUnscheduleJobs_Success`